template/components/ShareButton.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <a :href="shareURL" @click.prevent="share()" target="_blank"
+     :class="[ isButton ? `btn btn-${networkName}` : '' ]">
+    <slot>
+      <span>Share on {{ network }}</span>
+      <img v-if="networkName === 'twitter'"
```

Comment:
`v-if`s to handle asset inclusion issues. I'm happy to update this to a better Vue solution if there's one we like more or add the images in `/static`.